### PR TITLE
Add 4-Player Adapter documentation, adapted from Dan Docs.

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

**Comment by ISSOtm** [comment]
I'm not sure about whether this is in the scope of Pan Docs, given that this accessory is quite rare. Discussion on that in #16, though.

File: src/Four_Player_Adapter.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+transfer data (1KB/s), while setting it to 0xFF runs it close to the slowest
+speed (2042 bits-per-second).
+
+SIZE sets the length of packets exchanged between all Game Boys. Nothing fancy,
+just the number of bytes in each packet. It probably shouldn't be set to zero.
+
+### Transmission Phase
+
+When the master Game Boy (Player 1) is ready, it should send 4 bytes
+(`0xAA 0xAA 0xAA 0xAA`). (TODO: Are 4 bytes actually required?) This alerts
```
Comment (avivace) [inline src/Four_Player_Adapter.md:119]: I'm not sure TODOs should be scattered around like this

File: src/Four_Player_Adapter.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+with garbage data that is based on output the master Game Boy had sent during
+the ping phase. At this time, it is recommended to ignore the earliest packets
+received, however, it is safe to start putting new, relevant data into the
+buffer.
+
+### Restarting Ping Phase
+
+It's possible to restart the ping phase while operating in the transmission
+phase. To do so, the master Game Boy should send 4 or more bytes
+(`0xFF 0xFF 0xFF 0xFF`). (TODO: It's possible fewer 0xFF bytes need to be sent,
```
Comment (avivace) [inline src/Four_Player_Adapter.md:193]: same as previous comment

**Review by avivace — CHANGES_REQUESTED** [review]
LGTM and it definitely fits the scope of the accessories chapter of Pan Docs. My only worry here is how to keep up with upstream, since this adds an additional burden to sync with Dan Docs if it's a subject actively being investigated.

File: src/Four_Player_Adapter.md (hunk 1 above)
Comment (asiekierka) [reply src/Four_Player_Adapter.md:119]: Fair; what would you propose instead?

**Comment by avivace** [comment]
As we didn't get any constructive progress in the linked issues, I'm not gonna hold this any longer and merge it, I'll just fix the "todos"

**Comment by avivace** [comment]
TODOs removed from the text and added as Issues

**Pull request merged**